src/main.ts

```diff
@@ -196,4 +200,55 @@ async function addWorkflowSummary(msg: Message): Promise<void> {
   await summary.write();
 }
 
+async function sendSlackNotifications(
+  inputs: Inputs,
+  msg?: Message,
+): Promise<void> {
+  if (!inputs.slackWebhook) {
+    core.debug("Skipping Slack notification (no webhook)");
+    return;
+  }
+
+  // Use presence of msg to know if we're sending failure or success
```

Comment:
I'm on the fence about this. I did it in its own commit, so we can revert back to passing all the arguments if we prefer that.

Reply:
I'm good with this